test/unit/component/parser/json/BpmnJsonParser.label.bounds.test.ts

```diff
@@ -58,6 +58,7 @@ describe('parse bpmn as json for label bounds', () => {
                 BPMNLabel: {
                   id: 'label_id',
                   Bounds: { x: 25, y: 26, width: 27, height: 28 },
+                  isExpanded: true,
```

Comment:
:question: why not removing this addition and the element order in the BPMN element array triggering the test? The tests passed locally without them.
`isExpanding` is for call activity and subprocess, not for all activities. Except if you want to test that there is no parsing errors if the json model contains extra fields (which is not the purpose of this PR :smiley_cat: ), this should be removed IMHO

Reply:
I added some scpecific tests for call Activity, and after remove them in order to come back to the previous tests,. Maybe I forgot some adding 😉